WordPress/Classes/ViewRelated/Blog/JetpackSettingsViewController.swift

```diff
@@ -0,0 +1,470 @@
+import Foundation
+import CocoaLumberjack
+import WordPressShared
+
+
+/// The purpose of this class is to render and modify the Jetpack Settings associated to a site.
+///
+open class JetpackSettingsViewController: UITableViewController {
```

Comment:
Should this be `JetpackSecuritySettingsViewController`? Or are we going to put non-security Jetpack stuff in there as well?

Reply:
Yes, it can be `JetpackSecuritySettingsViewController`, at least for now I don't think we will be adding anything non security related, and the UI title is Security.